src/sparql/qc/mondo/qc-illegal-prefix-on-xref.sparql

```diff
@@ -22,7 +22,6 @@ WHERE
   }
   FILTER( STRBEFORE(str(?value),":") not in (
       "CSP",
-      "DERMO",
```

Comment:
I see I see. First I've heard of [DermO](https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-016-0085-x).

I we removing it because we just don't want to incorporate DermO at this time, or is it just a bad xref?

Reply:
I dont know what it is exactly, but since the xref is the only one from DERMO, and this is a hyperspecific disease (Herxheimer disease), I think it does not belong into Mondo until there is a proper strategy for it in place.

Reply:
Thanks for taking a look!

Reply:
Thanks for the explanation about this change.